ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/DataStructureUtil.java

```diff
@@ -302,7 +302,7 @@ public static BeaconBlock randomBeaconBlock(long slotNum) {
 
     for (int i = 0; i < numDeposits; i++) {
       DepositInput deposit_input =
-          new DepositInput(BLSPublicKey.empty(), Bytes32.ZERO, BLSSignature.empty());
+          new DepositInput(BLSPublicKey.random(), Bytes32.ZERO, BLSSignature.empty());
```

Comment:
This is a good change, but it causes one problem.  Because the MockP2PNetwork AND onChainStart both have to arrive at the same answer for the initialBeaconState we need to seed this random BLSPublicKey with entropy so that they both arrive at the same initial state